debugview/plugin.cpp

```diff
@@ -507,50 +434,9 @@ class debugview : public threadloop {
 
 		colorUniform = glGetUniformLocation(demoShaderProgram, "u_color");
 
-		groundObject.init(vertexPosAttr,
-			vertexNormalAttr,
-			colorUniform,
-			Ground_plane_NUM_TRIANGLES,
-			&(Ground_Plane_vertex_data[0]),
-			&(Ground_Plane_normal_data[0]),
-			GL_STATIC_DRAW
-		);
-
-		waterObject.init(vertexPosAttr,
-			vertexNormalAttr,
-			colorUniform,
-			Water_plane001_NUM_TRIANGLES,
-			&(Water_Plane001_vertex_data[0]),
-			&(Water_Plane001_normal_data[0]),
-			GL_STATIC_DRAW
-		);
-
-		treesObject.init(vertexPosAttr,
-			vertexNormalAttr,
-			colorUniform,
-			Trees_cone_NUM_TRIANGLES,
-			&(Trees_Cone_vertex_data[0]),
-			&(Trees_Cone_normal_data[0]),
-			GL_STATIC_DRAW
-		);
-
-		rocksObject.init(vertexPosAttr,
-			vertexNormalAttr,
-			colorUniform,
-			Rocks_plane002_NUM_TRIANGLES,
-			&(Rocks_Plane002_vertex_data[0]),
-			&(Rocks_Plane002_normal_data[0]),
-			GL_STATIC_DRAW
-		);
-
-		headsetObject.init(vertexPosAttr,
-			vertexNormalAttr,
-			colorUniform,
-			headset_NUM_TRIANGLES,
-			&(headset_vertex_data[0]),
-			&(headset_normal_data[0]),
-			GL_DYNAMIC_DRAW
-		);
+		// Load/initialize the demo scene.
+		demoscene = new ObjScene("demo2.obj");
```

Comment:
Minor: why not just `demo.obj`? Does that exist already? If you do decide to change this, remember to change it in `gldemo/plugin.cpp` as well.

Reply:
Woops, yes, should be just demo.obj; I used to have multiple for testing/evaluation.

Reply:
Fixed, also renamed to `scene.obj`.